docs/docs/docker/tagging-docker-images.mdx

```diff
@@ -130,9 +130,10 @@ docker_image(
 
 The default placeholders are:
 
-- `{directory}`: The directory the docker_image's BUILD file is in.
-- `{parent_directory}`: The parent directory of `{directory}`.
-- `{name}`: The name of the docker_image target.
+- `{name}`: The name of the `docker_image` target.
+- `{directory}`: The folder name of the docker_image's BUILD file.
+- `{parent_directory}`: The parent folder name of `{directory}`.
+- `{full_directory}`: The full path folder to the BUILD file.
```

Comment:
sorry to be nit picky, but this reads a little strange to me..
does this work for you?
```suggestion
- `{full_directory}`: The full path to the BUILD file.
```